sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1673,8 +1677,14 @@ void CAFMaker::produce(art::Event& evt) noexcept {
         FillTrackMCS(*thisTrack[0], trajectoryMCS, rec.reco.trk.back());
         FillTrackRangeP(*thisTrack[0], rangePs, rec.reco.trk.back());
 
+        art::Ptr<anab::T0> thisPFPT0;
+        if (fmPFPT0.isValid()) {
+          if (!fmPFPT0.at(iPart).empty()) {
+            thisPFPT0 = fmPFPT0.at(iPart).at(0);
```

Comment:
From the code conventions (and if I'm properly understanding what is returned here) I think this should be using a FindOneP or something that explicitly assumes there will only be one associated T0?

Reply:
Fixed.